airflow/models/renderedtifields.py

```diff
@@ -153,6 +154,23 @@ def _redact(self):
         for field, rendered in self.rendered_fields.items():
             self.rendered_fields[field] = redact(rendered, field)
 
+    @classmethod
```

Comment:
A thought not totally tied to this PR. Should the internal API calls use `staticmethod` instead, since they never use `cls`? Or _can_ they use `cls`? They should if so.